srcpkgs/ffmpeg6/template

```diff
@@ -196,7 +196,7 @@ ffmpeg6-devel_package() {
 	short_desc+=" - development files"
 	conflicts="ffmpeg-devel"
 	replaces="ffmpeg-devel>=0"
-	if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	if [[ "$XBPS_TARGET_MACHINE" = "i686"* ]]; then
```

Comment:
As in the other template, please use a case block rather than the bash test operator here.